### DISCUSSION TITLE
Any plan for the Avalonia rewrite?

### DISCUSSION THREAD

**Discussion by martinrhan** [stride3d/stride#2122] [General]
An desktop application for serious production, no matter which kind of work it does, is never a complete application if is don't have a fully featured plugin support system. Without that the application's capability is significantly limited.
I have been watching Avalonia rewrite for long time. The only change was launcher rewrite that was done a few weeks ago.
I believe that the dev team should put Avalonia rewrite into higher priority. Do they have any plan on it? For example, by what time is it going to be done?

**Comment by IXLLEGACYIXL** [comment]
in Stride , which is community-driven, there isn't a specific development team assigned, unlike what you might find in Unity or Godot.
 Instead, it relies on people contributing in their free time.  It's like a team effort where everyone plays a part.

Manio has started a Proof of Concept for a plugin system, but it's not complete yet. If you're interested, you can contribute to the implementation of the plugin system.

**Comment by tebjan** [comment]
It is mentioned in the 4.2 Release Blog post: [stride3d.net/blog/announcing-stride-4-2-in-dotnet-8/](https://www.stride3d.net/blog/announcing-stride-4-2-in-dotnet-8/)
